openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -172,7 +175,7 @@ const TeamDetailsV1 = ({
         teamUserPagin,
         isGroupType,
         isOrganization,
-        searchTerm ? table.length : undefined
+        table.length
```

Comment:
Is this change intentional?

Reply:
Yes it is. Just showing the count of teams in table.